### PR TITLE
address some a11y stuff and a minor footer nit

### PR DESCRIPTION
**Pull request by rossPatton**
this PR gets us to a 100 accessibility score via lighthouse, with only minor changes in a few spots

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -2,18 +2,20 @@
   <div class="contain">
     <img
       alt=""
+      role="presentation"
```
Comment (rossPatton) [inline _includes/footer.html:5]: logos and stuff like that dont need to be screen readable, and are in fact pretty annoying

File: _includes/footer.html [hunk 2]
```diff
@@ -2,18 +2,20 @@
...
     />
-    <ul class="d:flex flex-wrap d:flex-row justify-center list-style-none">
+    <ul class="d:flex flex-wrap d:flex-row list-style-none">
       {% for link in site.links %}
         <li class="d:c-1/3 marg-b-3 d:marg-r-3">
           {% if link.newTab %}
             <a
-              nofollow
-              noreferrer
+              class="black-important"
```
Comment (rossPatton) [inline _includes/footer.html:17]: normally red on white links are red on grey here, which is a contrast problem. black fits pass

File: _includes/footer.html [hunk 3]
```diff
@@ -2,18 +2,20 @@
...
+              rel="nofollow noopener noreferrer"
```
Comment (rossPatton) [inline _includes/footer.html:18]: i was too used to react and did this wrong before - this is how it's actually supposed to look

File: _includes/header.html [hunk 4]
```diff
@@ -83,12 +86,12 @@ <h1 class="title">
...
         </ul>
-      </div>
+      </nav>
 
-      <div class="flex flex-row ai-ctr rel d:hide">
+      <nav class="flex flex-row ai-ctr rel d:hide">
         <button class="flex" id="navBtn" role="button">
           <img
-            alt=""
+            alt="Click to render sub-page navigation"
```
Comment (rossPatton) [inline _includes/header.html:94]: so, modals and popups like this are very problematic for screen readers and this doesn't actually solve the problem, but it does help a tiny bit and passes the audit

ideally we'd want to focus the screen reader onto the newly opened modal, and at the same time hide everything else until it's closed, and then return the user back to where they were

which is obviously a huge pain on a small jekyll site - so im gonna punt that for now

File: _includes/main.scss [hunk 5]
```diff
@@ -229,7 +237,7 @@ ul {
 
 .d\:c-1\/3 {
   @include desktop {
-    flex: 1 1 calc(33.33% - 1rem);
+    flex: 0 1 calc(33.33% - 1rem);
```
Comment (rossPatton) [inline _includes/main.scss:240]: this just forces the columns in the footer to always be 1/3 on desktop

the collective actions link that was added took up 2/3s which causes it to be out of alignment

File: _includes/header.html (hunk 4 above)
Comment (shushugah) [reply _includes/header.html:94]: The other fix, can be to remove this modal and convert it to a separate page altogether, but this adds Alt Text in the meanwhile

**Review by shushugah — APPROVED** [review]
Nice!

**Pull request merged**